### PR TITLE
Add Lad

### PR DESCRIPTION
**Pull request by niftylettuce**
**By submitting this pull request, I promise I have read the [contribution guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖**

⬆⬆⬆⬆⬆⬆⬆⬆⬆⬆

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -403,6 +403,7 @@
 
 ### Web frameworks
 
+- [Lad](https://github.com/ladjs/lad) - Framework for full-stack apps and API's. Bundles a web server ([@ladjs/web](https://github.com/ladjs/web)), API server ([@ladjs/api](https://github.com/ladjs/api)), job scheduler ([bree](https://github.com/breejs/bree)), and proxy server ([@ladjs/proxy](https://github.com/ladjs/proxy)) with a full-stack MVC framework. Made by former Express TC and Koa team member [@niftylettuce](https://github.com/niftylettuce).
```
Comment (sindresorhus) [inline readme.md:406]: Should be added to the end. The description is too verbose.

**Comment by niftylettuce** [comment]
@sindresorhus I simplified it down to one short sentence!

**Comment by sindresorhus** [comment]
> Should be added to the end.

**Comment by niftylettuce** [comment]
Will do one sec

**Review by niftylettuce — COMMENTED** [review]
@sindresorhus done!

**Pull request merged**